scenes/quests/story_quests/verso/1_verso_combat_anger/verso_combat_components/verso_target_spriteframes.tres

```diff
@@ -0,0 +1,39 @@
+[gd_resource type="SpriteFrames" load_steps=6 format=3 uid="uid://cw0d60uuy2b8"]
+
+[ext_resource type="Texture2D" uid="uid://do4tec8oedbcx" path="res://scenes/quests/story_quests/template/2_template_combat/template_combat_components/template_target.png" id="1_s8lii"]
```

Comment:
This references the image in the template folder directly. I think you should either:

- Delete `verso_target.png` (since it is unused); or
- Update this to refer to `verso_target.png`